src/operator/nn/convolution.cc

```diff
@@ -60,8 +60,12 @@ static void ConvolutionComputeExCPU(const nnvm::NodeAttrs& attrs,
   const ConvolutionParam& params = nnvm::get<ConvolutionParam>(attrs.parsed);
   if (SupportMKLDNNConv(params, inputs[0])) {
     MKLDNN_OPCHECK_INIT(false, outputs.size(), inputs, outputs);
-    const auto mkldnn_inputs = GetMKLDNNInputArray(inputs);
-    MKLDNNConvolutionForward(attrs, ctx, mkldnn_inputs, req, outputs);
+    if (CheckMKLDNNInputArrayIsView(inputs)) {
```

Comment:
Make line 63 - 67 into a helper function, like
```
MKLDNNRun(MKLDNNConvolutionForward, attrs, ctx, inputs, req, outputs);
```
Then we can use this for all operators.